circle.yml

```diff
@@ -1,6 +1,11 @@
 machine:
   node:
     version: 6.7.0
+dependencies:
+  pre:
+    - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
versioning much? I think we could pull from https://github.com/yarnpkg/yarn/releases if we wanted to pin it and make updates their own PRs